CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 ### Fixed
 - export of customer group specific prices
 
+## [2.9.91] - 2020-06-24
```

Comment:
Please move up, change date so everything under "unreleased" will be part of the release

Reply:
done